### PR TITLE
doubles slime feed stats 

### LINKED ISSUE #8854: [SUGG] Xenobiology start-up buff
[Title]: Xenobiology start-up buff

[Implementation]: # Instead of the two gray slimes xenobio starts with it should start with ALPHA / OMEGA slimes (temp names or come up with something better). Alpha would split into a orange, purple and two gray slime, while Omega would split into blue, metal and two gray slimes. Giving you one tier 1 slime after the first split.

[Why]: # So many times i have tried to do xenobiology and i spent twenty minutes before i got tier 3 slimes, sometimes not even that high. As a traitor this is big gamble and not really worth doing nearly ever since the time you spend there you could killed the station over 5 times. Rounds don't last long as here as on /tg/ so to remove the start RNG it would speed it up and bring it up to par to hippie round speeds.

[Resources]: # Could use just the normal gray slime sprites for them

@HippieStation/maintainers

### PR DESCRIPTION
**Pull request by ghost**
[Guidelines]: it is modularized. #8854 
Closes #8854 

[Changelogs]: #
:cl: zamolxius
tweak: tweaked slime damage and nutrition gain rate to be doubled
balance: rebalanced slime feeding
/:cl:

### CODE REVIEW

**Comment by Carbonhell** [comment]
Xenobiology is already incredibly op due to the crossbreeding mechanics. I am unsure about this.

**Comment by ghost** [comment]
problem is,  nobody ever reaches that endgame content due to our short rounds.  this should double nutrient gain per second,  and halve the slime feeding timegate

**Comment by Manly-Dwarf** [comment]
`Gaylard laughs in the distance`

**Comment by Eagle-Eyess** [comment]
Also unsure about it but I do agree that it is too slow as it is so +1 for CHANGE

**Comment by Carbonhell** [comment]
Hello GMs and HA, please vote
Anyway fuck it +1, at worst we revert

**Review by ghost — APPROVED** [review]
mirroring procs is ugly but probably the best way to handle it here

File: hippiestation/code/modules/mob/living/simple_animal/slime/life.dm [hunk 1]
```diff
@@ -0,0 +1,60 @@
+/mob/living/simple_animal/slime/handle_feeding() //makes slimes gain nutrition faster to buff them for our shorter rounds
```
Comment (Carbonhell) [inline hippiestation/code/modules/mob/living/simple_animal/slime/life.dm:1]: which part of the main proc did you actually edit?is mirroring the whole proc required?

**Comment by Carbonhell** [comment]
Stale.

**Pull request closed without merging**